lib/views/teach_bpmf_view.dart

```diff
@@ -0,0 +1,421 @@
+import 'package:flutter/material.dart';
+import 'package:flutter/services.dart' show rootBundle;
+import 'package:flutter_tts/flutter_tts.dart';
+import 'package:fluttertoast/fluttertoast.dart';
+import 'package:ltrc/data/models/word_model.dart';
+import 'package:ltrc/data/models/unit_model.dart';
+import 'package:ltrc/data/providers/word_provider.dart';
+import 'package:ltrc/extensions.dart';
+import 'package:ltrc/widgets/teach_word/tab_bar_view.dart';
+import 'package:ltrc/widgets/teach_word/stroke_order_animator.dart';
+import 'package:ltrc/widgets/teach_word/stroke_order_animation_controller.dart';
+import 'package:provider/provider.dart';
+
+class TeachBopomoView extends StatefulWidget {
+  final String char;
+  final int unitId;
+  final String unitTitle;
+  final bool isBpmf;
+
+  const TeachBopomoView({
+    super.key,
+    required this.char,
+    required this.unitId,
+    required this.unitTitle,
+    required this.isBpmf,
+  });
+
+  @override
+  State<TeachBopomoView> createState() => _TeachBopomoViewState();
+}
+
+class _TeachBopomoViewState extends State<TeachBopomoView>
+    with TickerProviderStateMixin {
+  late StrokeOrderAnimationController _strokeOrderAnimationControllers;
+  late TabController _tabController;
+  FlutterTts ftts = FlutterTts();
+  late Word wordd;
```

Comment:
Is it a typo? Please use a clear variable name to specify the function of word, so that other engineers can understand it. Ex. newWord.

Reply:
that's for testing right now (getWord), i will delete it or change name in ithe next pr